Build failure: _64garm

### Steps To Reproduce

Steps to reproduce the behavior:
1. have `_64gram` in configuration file
2. update

### Build log

```Fish
╭──╴ on NixOS 24.11.0 ❄️  
┆ ~ 
╰─> home-manager switch --flake /etc/nixos/.#hand7s                                                                                                                            was 2m51s ❲✗ ❳ at ❗ 00:01 
error: builder for '/nix/store/dkwf5hgaxyg3fbdli1hf37axrd9a2d8h-64gram-1.1.31.drv' failed with exit code 1;
       last 10 log lines:
       >       |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >   254 |         return FakeAudioDeviceModuleImpl::Create(task_factory, std::move(*boxed_renderer), std::move(*boxed_recorder), options);
       >       |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       >   255 |       };
       >       |       ~
       > At global scope:
       > cc1plus: note: unrecognized command-line option '-Wno-unqualified-std-cast-call' may have been intended to silence earlier diagnostics
       > cc1plus: note: unrecognized command-line option '-Wno-ambiguous-reversed-operator' may have been intended to silence earlier diagnostics
       > cc1plus: note: unrecognized command-line option '-Wno-deprecated-volatile' may have been intended to silence earlier diagnostics
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/dkwf5hgaxyg3fbdli1hf37axrd9a2d8h-64gram-1.1.31.drv'.
error: 1 dependencies of derivation '/nix/store/a37mkwp12rrh7ifvym633gxj4cczrjd6-64gram-1.1.31-fish-completions.drv' failed to build
error: 1 dependencies of derivation '/nix/store/iw23x736f0mkwsn4aph90p933l851gb4-home-manager-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/f0wdd58d5005v1z7q3x42q8w6qs5d4x2-man-paths.drv' failed to build
error: 1 dependencies of derivation '/nix/store/lm8fjyii9775qwj1i72zll308cvn0a4q-home-manager-generation.drv' failed to build
```

### Additional context

i just tried to update my nixos and hm, and got this :D

### Notify maintainers

@clot27 
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```Bash

╭──╴ on NixOS 24.11.0 ❄️  
┆ ~ 
╰─> nix-shell -p nix-info --run "nix-info -m"                                                                                                                                            ❲✓ ❳ at ❗ 00:08 
 - system: `"x86_64-linux"`
 - host os: `Linux 6.10.4-cachyos, NixOS, 24.11 (Vicuna), 24.11.20240811.a58bc8a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.5`
 - channels(root): `"home-manager, nixos"`
 - nixpkgs: `/nix/store/h60m1fwahjd2mv6gsg77ji3vb4gpj4dk-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/_6/_64gram/package.nix
+++ b/pkgs/by-name/_6/_64gram/package.nix
@@ -1,13 +1,14 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, fetchpatch
 , telegram-desktop
 , nix-update-script
 }:
 
 telegram-desktop.overrideAttrs (old: rec {
   pname = "64gram";
-  version = "1.1.31";
+  version = "1.1.34";
 
   src = fetchFromGitHub {
     owner = "TDesktop-x64";
@@ -15,8 +16,20 @@ telegram-desktop.overrideAttrs (old: rec {
     rev = "v${version}";
 
     fetchSubmodules = true;
-    hash = "sha256-xYCousLXV9TeQjDNiXkEMbTiiuusLc7Ib2xHkMYBD1M=";
+    hash = "sha256-DbHbhkInZi8B/0fvbCWutN4noEv7MZ+C5Eg7g+89Moo=";
   };
+
+  patches = (old.patches or []) ++ [
+    (fetchpatch {
+      url = "https://github.com/TDesktop-x64/tdesktop/commit/c996ccc1561aed089c8b596f6ab3844335bbf1df.patch";
+      revert = true;
+      hash = "sha256-Hz7BXl5z4owe31l9Je3QOXT8FAyKcbsXsKjGfCmXhzE=";
+    })
+  ];
+
+  cmakeFlags = (old.cmakeFlags or []) ++ [
+    (lib.cmakeBool "disable_autoupdate" true)
+  ];
 
   passthru.updateScript = nix-update-script {};
 

